CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# v1.25.0
+
+## :zap: Nortable Changes
```

Comment:
```suggestion
## :zap: Notable Changes
```

I found the same typo in the v1.24.0 section. Sorry, I overlooked it in the last PR.